go/pkg/db/queries/identity_find_by_id.sql

```diff
@@ -1,2 +1,6 @@
 -- name: FindIdentityByID :one
```

Comment:
let's rename this to just `FindIdentity` and also update the filename for consistency